app/views/admin/reports/_fallback_display.html.haml

```diff
@@ -0,0 +1,44 @@
+.download.hidden
+  = link_to t("admin.reports.download.button"), file_url, target: "_blank", class: "button icon icon-file"
+
+:javascript
```

Comment:
I don't think it's a hard rule, but I think in general we should store javascript in javascript files. Advantages are being able to lint it and test it, and it's probably easier to find. 

I guess I would have tried making this a Stimulus controller. Although that would make my suggestion about mocking the timeout value harder...

Reply:
Good points. I'm not sure about the "easier to find" though. Yes, I would look in the assets for JS but I would first need to look into the view to know its name. And then the code is already there without looking somewhere else. I also like to bundle things together that are dependent on each other. :shrug: 

I think if this script becomes any more complex, moving it is a good idea.